Library/Contracts.ts

```diff
@@ -10,6 +10,7 @@ export module Contracts {
         SQL = 0,
         Http = 1,
         Other = 2,
+        ApplicationInsights = 3,
```

Comment:
Isn't this part of the public API, that is defining the valid values for the `dependencyKind` parameter to [`trackDependency`](https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/Library/Client.ts#L148)?